ydb/tools/ydbd_slice/ya.make

```diff
@@ -23,7 +23,6 @@ PY_SRCS(
 PEERDIR(
     ydb/tools/cfg
     ydb/public/sdk/python
-    ydb/public/sdk/python/enable_v3_new_behavior
```

Comment:
Some tests were broken, so I decided to remove thus peerdir because it seems meaningless in ydbd_slice